### PR TITLE
Review of To do list App

### PR DESCRIPTION
**Pull request by Efiamotu-1**
## Features Added

>- cloned version of my previous todo list
>- webpack 
>- ES6 modules

### CODE REVIEW

**Comment by limavfabio** [comment]
Good job so far Efiamotu! Your project is looking very nice!

But I think that there are still some things that can be improved, I will leave suggestions in the next comments

File: src/modules/display.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        }
+
+        return {
+          description: todo.description,
+          completed: todo.completed,
+          index: todo.index,
+        };
+      });
+      localStorage.setItem('todos', JSON.stringify(data.todos));
+      // e.target.parentElement.parentElement.remove();
```
Comment (limavfabio) [inline src/modules/display.js]: Inline comments of code are considered bad practice, I think your code would look more professional if you removed them. And you don't have to worry about losing those previous implementations, they will always be available in your git history

**Comment by limavfabio** [comment]
Maybe it would also help to have a button for adding the tasks alongside being able to press the ENTER key, as to improve user experience.

**Review by limavfabio — COMMENTED** [review]
Maybe it would also help to have a button for adding the tasks alongside being able to press the ENTER key, as to improve user experience.

File: src/modules/display.js (hunk 1 above)
Comment (Efiamotu-1) [reply src/modules/display.js]: Thank you very much for spotting this, i will make the required changes as soon as possible

**Review by limavfabio — APPROVED** [review]
Very well done Efiamotu! Your project is approved, it's time to merge it! 💪🏽 🎉

**Pull request merged**